### DISCUSSION TITLE
Migrate from InfluxDB ends with error message

### DISCUSSION THREAD

**Discussion by madface303** [VictoriaMetrics-Community/homeassistant-addon-victoriametrics#15] [Q&A] [answered]
I am trying to import my influxdb data to victoria metrics, using vmctl as mentioned in the docs, the last version from the vm homepage.
`./vmctl-prod influx --influx-user homeassistant --influx-password mysupersecretpw --influx-database homeassistant`

This should copy the autogen policy data to vm (at least i would think that). It gives me the following error:

```
2023/01/01 15:03:52 Exploring scheme for database "homeassistant"
2023/01/01 15:03:52 fetching fields: command: "show field keys"; database: "homeassistant"; retention: "autogen"
2023/01/01 15:03:52 found 80 fields; skipped 99 non-numeric fields
2023/01/01 15:03:52 fetching series: command: "show series"; database: "homeassistant"; retention: "autogen"
2023/01/01 15:03:52 found 404 series
2023/01/01 15:03:52 explore query failed: can't find field keys for measurement "ETHUSD"
```
Any idea what i'm doing wrong?

**Comment by fuslwusl** [comment, marked as answer]
Just contact victoria metrics for this kind of support.
